keyboards/noxary/x268/x268.c

```diff
@@ -15,35 +15,29 @@
  */
 #include "x268.h"
 
+// Optional override functions below.
+// You can leave any or all of these undefined.
+// These are only required if you want to perform custom actions.
+
 void matrix_init_kb(void) {
 	// put your keyboard start-up code here
 	// runs once when the firmware starts up
-
+	// setPinOutput(B1);
 	matrix_init_user();
 }
 
-void matrix_scan_kb(void) {
-	// put your looping keyboard code here
-	// runs every cycle (a lot)
-
-	matrix_scan_user();
-}
-
-bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
-	// put your per-action keyboard code here
-	// runs for every action, just before processing by the firmware
-
-	return process_record_user(keycode, record);
-}
+// bool led_update_kb(led_t led_state) {
+//     if(led_update_user(led_state)) {
+//         writePin(B1, led_state.caps_lock);
+//     }
+//     return true;
+// }
 
 void led_set_kb(uint8_t usb_led) {
 	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
-
 	led_set_user(usb_led);
 }
-
 void led_set_user(uint8_t usb_led) {
```

Comment:
This function should be updated to `led_update_kb()`

Reply:
Updated function